### PR TITLE
jupyter nb - uuid1 to uuid4

### PR DESCRIPTION
**Pull request by fragmuffin**
fix for: #76 

"a 1 character bugfix... I'LL TAKE IT!"

### CODE REVIEW

File: cadquery/occ_impl/jupyter_tools.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
 def add_x3d_boilerplate(src, height=400, center=(0,0,0), d=(0,0,15), fov=FOV, rot='{} {} {} {} '.format(*ROT)):
 
     return BOILERPLATE.format(src=src,
-                              id=uuid1(),
```
[the remark this answers is not in the document]
Comment (fragmuffin) [reply cadquery/occ_impl/jupyter_tools.py]: :man_facepalming: oops! a 2 character bugfix then?
hmm. that means there aren't any tests for those functions.

**Comment by fragmuffin** [comment]
That's better; tests show the failed import picked up by @adam-urbanczyk :+1:

File: cadquery/occ_impl/jupyter_tools.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (fragmuffin) [reply cadquery/occ_impl/jupyter_tools.py]: fixed in: d55e22de1fd629fa57b7cebdb6acd9132b1471d4

**Comment by adam-urbanczyk** [comment]
@fragmuffin thanks for cleaning up my mess.

File: cadquery/occ_impl/jupyter_tools.py [hunk 2]
```diff
@@ -98,6 +98,6 @@ def x3d_display(shape,
         vec = quat*(vec) + c.wrapped
 
         # return boilerplate + Scene
-        return add_x3d_boilerplate(ElementTree.tostring(scene_tag),
+        return add_x3d_boilerplate(str(ElementTree.tostring(scene_tag).decode('utf-8')),
```
Comment (adam-urbanczyk) [inline cadquery/occ_impl/jupyter_tools.py]: str call seems not to be needed for python2.  _repr_html_ can return unicode on py2

Comment (fragmuffin) [reply cadquery/occ_impl/jupyter_tools.py]: good point ;), removed in ad6d0b0b9356efbfe6c3d95d91315e02995f6858

**Pull request merged**